Source/Libraries/GSF.Web/Shared/Scripts/gsf.web.client.js

```diff
@@ -136,7 +136,7 @@ function clearCachedCredentials(securedUrl, successCallback) {
         if (successCallback)
             xhr.onreadystatechange = function() {
                 if (this.readyState === XMLHttpRequest.DONE)
-                    successCallback(this.status === 401 || this.status === 403);
+                    successCallback(this.status === 200, this.responseText);
```

Comment:
Guess this was what was causing the issue. Glad you got this fixed.

I think this is first time I've tried this code review thing - it's pretty decent.

Reply:
Actually, this code was not causing any error. I only changed it because the new logout page returns a 200 for success. The old "clearCredentials" page (which was actually just `AuthTest`) would return a 401 after `securityHub.logout()`, which is what we were expecting here. The error we got was because we didn't call `securityHub.logout()` and therefore got a completely unrelated error because we were still authenticated.